gtests/net/mptcp/add_addr/add_addr_server.pkt

```diff
@@ -4,7 +4,7 @@
 
 // TODO(malsbat):
 +0     `ip mptcp endpoint flush`
-+0     `ip mptcp endpoint add 192.168.122.83 signal`
++0     `ip mptcp endpoint add 198.51.100.2 signal`
```

Comment:
I forgot to ask the question: why changing the IP?

Reply:
oh, that is a leftover of a test aiming to see whether parsing of "inet_addr" below was working for real. But I think we can keep this, it's even better because that address is not in the server's network nor in the packetdrill's one (at least, when run_all.py runs)

Reply:
I see, good point!

If you have to modify the commit, do not hesitate to add a comment here and/or in the commit message (or a dedicated commit :) )